adal/src/main/java/com/microsoft/aad/adal/ADALError.java

```diff
@@ -554,7 +554,7 @@
     
     private String mDescription;
 
-    private ADALError(String message) {
+    ADALError(String message) {
```

Comment:
Android studio says private constructor for enum does not make sense.

---
In reply to: [65638135](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/652#discussion_r65638135) [](ancestors = 65638135)